.github/workflows/delete-release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
       - name: Set output
         id: vars
-        run: echo "tag=${GITHUB_REF#refs/*/} >> $GITHUB_OUTPUT
+        run: echo "{tag}={${GITHUB_REF#refs/*/}}" >> $GITHUB_OUTPUT
```

Comment:
I guess this was broken? :D

Reply:
I think so as well, I think it would make sense to try out the modifications soon, hopefully they will work on a fork as well (have not checked yet).